packages/web/utils/assertion.ts

```diff
@@ -0,0 +1,3 @@
+export function isFunction(value: any): value is Function {
+  return typeof value === "function";
```

Comment:
I like the util since it's easier to read `isFunction` instead of `typeof value === "function"`. Also, it follows a functional paradigm making it easier to use in `pipe` or `compose` functions. I'm open to leave it as `typeof` though. Please let me know your thoughts.